config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'recipes#new', as: 'root'
+
+  resources :recipes, only: [:index, :new, :show, :create]
```

Comment:
I don't think that `new` is an appropriate name for the search action, since it's not creating a new recipe. Similarly, I don't think you need the `new` or `create` resources for `recipes` here.